pkg/clients/cloudsql/cloudsql.go

```diff
@@ -110,6 +110,7 @@ func GenerateDatabaseInstance(name string, in v1beta1.CloudSQLInstanceParameters
 				ExpirationTime: gcp.StringValue(val.ExpirationTime),
 				Name:           gcp.StringValue(val.Name),
 				Value:          gcp.StringValue(val.Value),
+				Kind:           "sql#aclEntry",
```

Comment:
Note: anywhere where we fully replace a struct when it is not `nil`, we need to make sure that we include all fields like this. I checked `CloudMemorystore`, `GKECluster`, and `NodePool` for similar issues and did not find any.

Reply:
`ACLEntry`. I think that might be a high level API resource that I was planning to implement a controller for during bucket stuff https://github.com/crossplaneio/stack-gcp/pull/137

Reply:
Though this PR is not the right place to do it right now. So, my comment is not blocking